### DISCUSSION TITLE
Processing of the 20240330.194110 1074 scan file with the extra 0.3nm wave offset.

### DISCUSSION THREAD

**Discussion by bberkeyU** [NCAR/ucomp-pipeline#269] [General]
We can recover a few 1074 nm from 20240330 (there is only one good 1074 nm file right now).

@bberkeyU says:

Is it worth the effort to try and process 20240330.194110.39.ucomp.1074.l0.fts / 20240330.194330.43.ucomp.1074.l0.fts (I apologize; I forgot which was flat and which was the coronal data)? These should have reasonable 1074 data just shifted ~0.30nm to the red based on the wavoff change for the search. The sun was active that day, and the 20-minute baseline between the two measurements might be scientifically interesting to some people.

Processing this will require a hack to subtract 0.3 nm from the reported wavelengths before we do any fitting. Our line spacing in the recipe was 0.12nm, so we didn't tune to our nominal middle 3 wavelengths but instead 1074.52, 1074.64, and 1074.76 (not the expected 1074.58, 1074.7, and 1074.82), which will probably mess up the mean/median files.
We could make a repair routine to modify the reported wavelength in the headers for the appropriate files.

About 20 minutes before the 20240330.201401 1074 image that is currently on the web, we searched for the lyot filter drift with a +0.30nm wavelength offset. This +0.3 offset data in 20240330.194110 contains the complete 1074 profile, but it is just 0.3nm to the blue.

@detoma @jburkepile  For the 1.1 run should we figure out how to process this file and make it available online? If we were to edit the L0 meta-data to subtract 0.3 from WAVOFF, WAVELNG, and  EXTNAME, the pipeline should be able to process this file without an issue.   We may run into a problem when we do the daily averages because the 194110 file tunings are .06nm blue of 201401, but if we get a good Gaussian fit, this may not be a problem.  

![20240330 194110 ucomp 1074 l1 p9 iquv all](https://github.com/NCAR/ucomp-pipeline/assets/22605316/ec769b3b-e16c-44c0-a515-eb8ad58665bc)

**Comment by detoma** [comment]
From @detoma
Even with the 0.30nm hack we will not have a point centered on the line, right?

I am not sure it is possible to properly process a file that does not have a tuning on the line center without major changes in the code. I will let Mike comment on that.

@bberkeyU
The two files have a 0.06nm tuning difference with the peak falling between two of the tunings in 194110. The 0.06 is double our average wave offset correction applied during 2021/2022. It maybe hard to process this file. But this file does have coronal signal on a day with active sun where we only made one measurement so is it worth trying?
